executor/merge_join_test.go

```diff
@@ -221,17 +222,17 @@ const plan3 = `[[TableScan_12 {
     "desc": "false"
 } ]]`
 
-func checkMergeAndRun(tk *testkit.TestKit, c *C, sql string) *testkit.Result {
+func checkMergeAndRun(tk *testkit.TestKit, t *testing.T, sql string) *testkit.Result {
 	explainedSQL := "explain format = 'brief' " + sql
 	result := tk.MustQuery(explainedSQL)
 	resultStr := fmt.Sprintf("%v", result.Rows())
 	if !strings.ContainsAny(resultStr, "MergeJoin") {
-		c.Error("Expected MergeJoin in plan.")
+		t.Error("Expected MergeJoin in plan.")
 	}
 	return tk.MustQuery(sql)
 }
 
-func checkPlanAndRun(tk *testkit.TestKit, c *C, plan string, sql string) *testkit.Result {
+func checkPlanAndRun(tk *testkit.TestKit, t *testing.T, plan string, sql string) *testkit.Result {
```

Comment:
Please either remove the `t *testing.T` argument or fix the commented out code that currently uses `c *C`.

Reply:
I'm sending a patch to address this.